index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Apple</title>
+  <link rel="shortcut icon" href="./images/apple-favicon.png" type="image/x-icon">
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+    integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css" />
+
+</head>
+
+<body>
+  <!-- Main Section -->
+  <main class="first-container">
+    <section class="main-stuff">
+      <div class="container">
+        <nav class="nav-links">
+          <ul>
+            <li><a href="#"><i class="fab fa-apple"></i></a></li>
+            <li><a href="#">Store</a></li>
+            <li><a href="#">Mac</a></li>
+            <li><a href="#">iPod</a></li>
+            <li><a href="#">iPhone</a></li>
+            <li><a href="#">iPad</a></li>
+            <li><a href="#">iTunes</a></li>
+            <li><a href="#">Support</a></li>
+            <li>
+              <form action="#" class="main-search">
+                <input type="search" name="search" />
+                <i class="fas fa-search fa-sm"></i>
+              </form>
+            </li>
+
+          </ul>
+        </nav>
```

Comment:
- [ ] Navbar should be wrapped around ```<header>``` HTML tag, and outside the ```<main>``` container. Example:
```
...
  <header>
    <nav>
      ... nav elements ...
    </nav>
  </header>

  <main>
    ... main content ...
  </main>
...
```